azure-pipelines.yml

```diff
@@ -31,3 +31,11 @@ steps:
     pip install pytest pytest-azurepipelines
     pytest
   displayName: 'pytest'
+
+- task: PublishCodeCoverageResults@1
```

Comment:
I don't think you need this task as `pytest-azurepipelines` already does it?

See https://pypi.org/project/pytest-azurepipelines/